src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -6,374 +6,49 @@ redirect_from:
-
-</Note>
-
-#### How is the Sentry Integration different from OAuth Apps?
-
-OAuth Applications allow an external developer to create an application which can authenticate as a Sentry user and take actions as the user within Sentry.
-
-Sentry Integration apps are similar to OAuth apps, except Sentry Integration apps act as an independent entity. There is currently no way for Sentry Integration apps to act on behalf of a user.
-
-#### How is the Sentry Integration different from personal Auth Tokens?
-
-Personal Auth Tokens are tokens a user can use to invoke APIs directly and have access to all the resources tied to that user. Sentry Integration Apps do not represent a single user, but rather make requests as "itself."
+## UI Components
 
-#### We use the Sentry API internally at our company. Should we switch to the Sentry Integration Platform?
+The Sentry Integration Platform provides the ability to add rich UI components to [sentry.io](https://sentry.io) itself through a  simple, declarative syntax. For more information, check out the [full UI Components documentation](/product/integrations/integration-platform/ui-components/).
 
-If you're looking for any of the new features below, we recommend you switch to the Sentry Integration Platform utilizing the [Internal Integrations](#internal-integrations).
+## Alerts
 
-**Permissions**
+You can make any integration available as an action in [issue alerts](/product/integrations/integration-platform/webhooks/#issue-alerts) and [metric alerts](/product/integrations/integration-platform/webhooks/#metric-alerts) by enabling the "Alert Rule Action" toggle. The integration will then show
+up as a service in the action section when creating or updating an alert rule:
 
-OAuth apps and Auth Tokens allow you to access Sentry as a specific user in your Sentry org. This means:
+![Dropdown menu of actions for an alert rule.](alert-rules.png)
 
-- If this user leaves the org or their permissions change, the integration can break. A Sentry Integrations App, on the other hand, is a separate, independent entity with its own permissions.
-- OAuth apps and Auth Tokens permit the user to access other Sentry organizations they belong to, so there is limited isolation.
-- Actions taken in Sentry will connect to this user rather than the integration/app. For example, if your alerting integration automatically assigns issues to a teammate, it will appear in the history as “Alice assigned Bob issue XYZ” rather than “&lt;Your Sentry Integration apps&gt; assigned Bob issue XYZ.”
+For your service to receive webhooks for alert rules, users must add to existing rules or create new ones that have `Send a notification via <your service>` as an action in the rule. Once that's set up, you'll start receiving webhook requests for triggered alerts. For more information about the request and payload, check out the [full Webhooks documentation](/product/integrations/integration-platform/webhooks/).
```

Comment:
I think you should add a sentence here to direct devs to Alert Rule Actions UI component if they need more specific information:

> If you'd like users to provide more app-specific information to route/handle these webhooks (e.g. assignees, teams, notification channels), consider implementing the [alert rule action UI component](#link)

Though my wording probably isn't great, feel free to change it!